userspace/libsinsp/container_engine/docker/podman.cpp

```diff
@@ -193,7 +199,7 @@ bool podman::resolve(sinsp_threadinfo *tinfo, bool query_os_for_missing_info)
 
 	if(!m_api_sock_can_exist.has_value())
 	{
-		m_api_sock_can_exist = can_api_sock_exist();
+		m_api_sock_can_exist = can_api_sock_exist(tinfo->m_inspector);
```

Comment:
I thought we should have leveraged `query_os_for_missing_info`, similar to other container `resolve` implementations, which is being passed as `m_inspector->is_live() || m_inspector->is_syscall_plugin()`, WDYT?
```
userspace/libsinsp/parsers.cpp:1359:                    m_inspector->m_container_manager.resolve_container(child_tinfo.get(), m_inspector->is_live() || m_inspector->is_syscall_plugin());
userspace/libsinsp/parsers.cpp:1909:            m_inspector->m_container_manager.resolve_container(child_tinfo.get(), m_inspector->is_live());
userspace/libsinsp/parsers.cpp:2184:            m_inspector->m_container_manager.resolve_container(evt->get_tinfo(), m_inspector->is_live() || m_inspector->is_syscall_plugin());
userspace/libsinsp/parsers.cpp:5460:            m_inspector->m_container_manager.resolve_container(evt->get_tinfo(), m_inspector->is_live() || m_inspector->is_syscall_plugin());
userspace/libsinsp/threadinfo.cpp:503:  m_inspector->m_container_manager.resolve_container(this, m_inspector->is_live() || m_inspector->is_syscall_plugin());
```

Reply:
Or even better, just 
```C
if (query_os_for_missing_info) m_api_sock_can_exist = can_api_sock_exist();
else _api_sock_can_exist = true;
```

Reply:
I like it! Thanks!

Reply:
Done